.github/workflows/release.yml

```diff
@@ -310,6 +310,21 @@ jobs:
           console.log("The canonical version of this Metabase", edition, "edition is", canonical_version);
 
           return canonical_version;
+    - name: Determine the docker short version tag
+      uses: actions/github-script@v7
+      id: short_canonical_version
+      with:
+        result-encoding: string
+        script: |
+          const edition = '${{ matrix.edition }}';
+
+          const canonical_version = "${{ steps.canonical_version.outputs.result }}";
+          const versionParts = canonical_version.split("."); // ["v1", "51", "1", "3"]
+          const shortCanonicalVersion = versionParts.slice(0, 2).join("."); // "v1.51"
+
+          console.log("The short canonical version of this Metabase", edition, "edition is", shortCanonicalVersion);
+
+          return shortCanonicalVersion;
```

Comment:
instead of doing this inline, we should create a function in `version-helpers.ts` that can be unit tested